server/src/test/java/org/elasticsearch/index/fielddata/ScriptDocValuesLongsTests.java

```diff
@@ -95,7 +95,7 @@ public void testDates() throws IOException {
                 assertEquals(dates[d][i], longs.getDates().get(i));
             }
 
```

Comment:
this test only wants to verify the list is frozen, doesnt care about the "value".